### PR TITLE
Improve code coverage

### PR DESCRIPTION
**Pull request by yimoj**

### CODE REVIEW

File: tests/integration/test_scan.py [hunk 1]
```diff
@@ -131,6 +131,41 @@ def _run_tests(op, n0, shape, dt, axis, out0, outtype):
...
+        (np.int16, np.float64),
+        (np.complex64, np.float64),
+        (np.float32, np.int64),
+    ],
+    ids=str,
+)
+@pytest.mark.parametrize(
+    "op",
+    [
+        pytest.param("cumsum", marks=pytest.mark.xfail),
```
Comment (bryevdv) [inline tests/integration/test_scan.py:146]: What exactly is the failure for `cumsum` specifically?

File: tests/integration/test_scan.py [hunk 2]
```diff
@@ -131,6 +131,41 @@ def _run_tests(op, n0, shape, dt, axis, out0, outtype):
...
+)
+def test_scan_out_dtype_mismatch(dtype, outtype, op):
+    # When CUNUMERIC_TEST=1:
+    # out_np: array([0., 0., 0., 0., 0., 0.])
+    # out_num:
+    # array([6.8983227e-310, 6.8983227e-310, 6.8983227e-310, 6.8983227e-310,
+    #        6.8983227e-310, 6.8983227e-310])
+    shape = (1, 2, 3)
+    out_shape = (6,)
+    arr = np.empty(shape)
```
Comment (bryevdv) [inline tests/integration/test_scan.py:161]: `empty` is a good choice for out-arrays, I might consider something more defined and repeatable for inputs

**Review by bryevdv — APPROVED** [review]
One suggestion otherwise LGTM

File: tests/integration/test_scan.py (hunk 2 above)
Comment (yimoj) [reply tests/integration/test_scan.py]: Good point, actually found more issues after changing the input.

File: tests/integration/test_scan.py (hunk 1 above)
Comment (yimoj) [reply tests/integration/test_scan.py:151]: Moved the explanation to a more relatable location.

**Pull request merged**